plugin/EstateList.php

```diff
@@ -397,9 +402,46 @@ public function estateIterator($modifier = EstateViewFieldModifierTypes::MODIFIE
 			$pEstateStatusLabel = $this->_pEnvironment->getEstateStatusLabel();
 			$recordModified['vermarktungsstatus'] = $pEstateStatusLabel->getLabel($recordRaw);
 		}
+		if ($this->_pWPOptionWrapper->getOption('onoffice-settings-title-and-description') == 0)
+		{
+			$recordModified['onoffice-settings-title-and-description'] = true;
+		}
+		$lang = Language::getDefault();
+		if ($this->_pWPOptionWrapper->getOption('onoffice-settings-title-and-description') == 1)
+		{
+			$listCustomField = [
```

Comment:
Thank you for refactoring this. Sadly, EstateList grows and grows but can hardly be tested. That's why this new block should be in its own class.